src/SoapCore.Tests/Serialization/Models.DataContract/ComplexModel2.cs

```diff
@@ -4,7 +4,7 @@
 namespace SoapCore.Tests.Serialization.Models.DataContract
 {
 	// same as ComplexModel1, just for testing
-	[DataContract]
+	[DataContract(Namespace = "http://SoapCore.Tests.Serialization.Models.DataContract.ComplexModel2")]
```

Comment:
This might unexpectedly change test coverage. What happens if in the scenario where the namespace is left undefined?

Reply:
No change (error) would happen to the input model with no namespace but I've added a test case to prove and enforce this.